# Generated by Django 3.0.4 on 2020-06-03 15:21

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('ATMStatus', '0015_auto_20200506_1512'),
    ]

    operations = [
        migrations.AlterField(
            model_name='atmlogincredentialsdetails',
            name='ATM_journal_password',
            field=models.CharField(blank=True, error_messages={'invalid': 'your custom error message'}, max_length=50, null=True),
        ),
        migrations.AlterField(
            model_name='atmlogincredentialsdetails',
            name='ATM_journal_user_name',
            field=models.CharField(blank=True, error_messages={'invalid': 'your custom error message'}, max_length=50, null=True),
        ),
        migrations.AlterField(
            model_name='atmlogincredentialsdetails',
            name='R_admin_password',
            field=models.CharField(blank=True, error_messages={'invalid': 'your custom error message'}, max_length=50, null=True),
        ),
        migrations.AlterField(
            model_name='atmlogincredentialsdetails',
            name='R_admin_user_name',
            field=models.CharField(blank=True, error_messages={'invalid': 'your custom error message'}, max_length=50, null=True),
        ),
        migrations.AlterField(
            model_name='atmlogincredentialsdetails',
            name='VNC_password',
            field=models.CharField(error_messages={'invalid': 'your custom error message'}, max_length=50),
        ),
    ]
